Content.Server/_DV/StationEvents/Events/PsionicNosebleedRule.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Server._DV.StationEvents.Components;
+using Content.Server.Body.Systems;
+using Content.Server.Psionics.Glimmer;
+using Content.Server.Station.Systems;
+using Content.Server.StationEvents.Events;
+using Content.Shared.Abilities.Psionics;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.Psionics.Glimmer;
+
+namespace Content.Server._DV.StationEvents.Events;
+
+public sealed class PsionicNosebleedRule : StationEventSystem<PsionicNosebleedRuleComponent>
+{
+    [Dependency] private readonly GlimmerSystem _glimmer = default!;
+    [Dependency] private readonly StationSystem _stationSystem = default!;
+    [Dependency] private readonly GlimmerReactiveSystem _glimmerReactiveSystem = default!;
+    [Dependency] private readonly MobStateSystem _mobStateSystem = default!;
+    [Dependency] private readonly BloodstreamSystem _bloodstreamSystem = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
```

Comment:
most of these are unused
also rename them to not have System in the field name its already obvious
_mob _bloodstream etc